test/minuteman_test.rb

```diff
@@ -211,3 +212,18 @@
     assert user.count("login:attempts").day.count == 3
   end
 end
+
+# scope "do action throught a user with different scopes" do
```

Comment:
Can you please add a note or remove? Is this pending?

Reply:
Removed it.